css/index.css

```diff
@@ -1,3 +1,282 @@
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+
+body {
+	padding-bottom: 40px;
+}
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+
+
+/* NAVIGATION */
+
+.container {
+	margin: 0 auto;
+	max-width: 1000px;
+}
+
+.navbar{
+	display: flex;
+} 
+
+ .navbar div {
+	margin-top: 40px;
+	margin-bottom: 40px;
+	padding-right: 20px;
+} 
+
+.navbar nav {
+	display: flex;
+	flex-direction: row;
+}
+
+.navbar nav a {
+	color: darkgray;
+	text-decoration: none;
+	padding: 20px;
+	font-weight: bold;
+	margin: 0 10px;
+}
+
+.logo {
+	margin-top: 20px;
+	margin-left: 80px;
+}
+
+
+@media(max-width: 800px) {
+
+    .container .navbar{
+        display: flex;
+        flex-direction: column-reverse;
+    } 
+
+    .container .logo {
+        margin-left: 0;
+        margin-top: 40px;
+        margin-bottom: 30px;
+    }
+
+    .container .navbar div {
+        justify-content: center;
+        margin: 0 auto;
+        padding-right: 0;
+    }
+}
+
+
+@media(max-width: 500px) {
+
+    .navbar nav {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    .navbar nav a {
+        justify-content: center;
+        /* width: 100%; */
+        padding: 10px 20px;
+        border-top: 1px solid black;
+    }
+
+    .navbar nav a:last-child {
+        border-bottom: 1px solid black;
+    }
+
+}
+
+
+/* SECTION ONE */
+
+.section-one {
+	display: flex;
+	justify-content: center;
+	margin: 30px 0;
+}
+
+.section-one h1 {
+	font-size: 80px;
+}
+
+.title {
+	text-align: center;
+	padding: 30px 100px;
+	margin: 20px 80px;
+	line-height: 35px;
+}
+
+
+
+@media(max-width: 800px) {
+    .large-img {
+        display: none;
+    } 
+}
+
+
+
+
+/* SECTION TWO */
+
+.content {
+	border-top: 1px solid black;
+	border-bottom: 1px solid black;
+}
+
+.section-two, .section-four {
+	display: flex;
+	justify-content: space-between;
+	padding: 40px 0;
+}
+
+.button {
+	text-decoration: none;
+	border: 1px solid black;
+	padding: 5px 40px;
+	color: black;
+	margin-top: 50px;
+}
+
+.section-two div, .section-four div {
+	margin-right: 20px;
+}
+
+
+@media(max-width: 800px) {
+    .content {
+        border-top: 2px solid black;
+        border-bottom: 2px solid black;
+        margin: 0 10px;
+    }
+
+}
+
+@media(max-width: 500px) {
```

Comment:
Try not using multiple media queries calling the same argument, it gets confusing when referencing your code later down the line